api/notd/token_staking_processor.py

```diff
@@ -0,0 +1,46 @@
+import datetime
+from typing import List
+from typing import Tuple
+from typing import Dict
+import sqlalchemy
+
+from notd.model import RetrievedTokenStaking
+from notd.model import CREEPZ_STAKING_CONTRACT
+from notd.store.retriever import Retriever
+from notd.store.schema import BlocksTable
+from notd.store.schema import TokenTransfersTable
+
+class TokenStakingProcessor:
+
+    def __init__(self, retriever: Retriever) -> None:
+        self.retriever = retriever
```

Comment:
i would expect the hardcoded list of staking addresses to be in this class somewhere